create-model

```diff
@@ -134,9 +153,9 @@ main () {
     search_replace "<@author>" "$AUTHOR"
     search_replace "<@email>" "$EMAIL"
     # Exceptions
-    sed -i "s/model_name/$model_name/g" setup.py docs/hack/docs.sh
-    sed -i "s/project/$project_name/g" Makefile
-    sed -i "s/model_name/$model_name/g" \
+    sed -i.bak "s/model_name/$model_name/g" setup.py docs/hack/docs.sh
```

Comment:
What `-i.bak` does? There is some side effect on Linux?

Reply:
According to Stack Overflow discussions, the iOS sed command must provide an extension to the backup file while it is optional in Unix. So, '.bak' is the choosen extension.